### PR TITLE
fix(react-google-adk): report missing stream bodies

### LINKED ISSUE #7690: react-google-adk: an event-stream response without a body throws a TypeError
## Problem

A successful ADK response with the expected `text/event-stream` content type but no response body throws an internal `Cannot read properties of null` TypeError.

## Reproduction

Return `new Response(null, { status: 200, headers: { "Content-Type": "text/event-stream" } })` from the configured fetch call and start consuming the ADK stream.

## Root cause

The response content type is validated, but `parseSSEResponse` uses `response.body!.getReader()` without validating that the body exists.

## Proposed fix

Validate the response body before creating the reader and throw a package-level ADK response error that explains the missing stream body.

## Verification

Add a focused client test asserting the stable error message for a successful event-stream response with a null body.

<a href="https://rupic.dev/s/s2.KjU4gPSfeF_oc29v5ya2wYK7l-1A-UQ49K5o_W_0StbP0BFctZrVly6SsCw?ref=github" target="_blank" rel="noopener noreferrer"><picture><source media="(prefers-color-scheme: dark)" srcset="https://rupic.dev/badges/track-dark-v1.svg">Track in Rupic</picture></a>

### PR DESCRIPTION
**Pull request by Kinfe123**
## Problem

A successful ADK response with the expected text/event-stream content type but no body throws an internal getReader TypeError instead of explaining the invalid response. Some non-standard fetch implementations expose the missing body as undefined rather than null.

Fixes #7690.

## Root cause

The stream parser assumed every successful event-stream response had a readable body.

## Change

Validate that the event-stream response has a truthy body before creating its reader. Both null and undefined bodies now report the same stable ADK response error.

## Verification

- Confirmed the null-body reproduction receives the raw getReader TypeError before the fix
- Added coverage for a non-standard response with an undefined body
- pnpm --filter @assistant-ui/react-google-adk exec vitest run src/AdkClient.test.ts --maxWorkers=2 (72 tests)
- pnpm --filter @assistant-ui/react-google-adk... build
- Focused oxlint, oxfmt, and git diff --check

## Public surface

@assistant-ui/react-google-adk error behavior changes for invalid successful responses. No exports, documentation, or generated API reference change. A patch changeset is included.

### CODE REVIEW

File: packages/react-google-adk/src/AdkClient.ts [hunk 1]
```diff
@@ -252,7 +252,10 @@ function messagesToProxyBody(
 }
 
 async function* parseSSEResponse(response: Response): AsyncGenerator<AdkEvent> {
-  const reader = response.body!.getReader();
+  if (response.body === null) {
```
[the remark this answers is not in the document]
Comment (Kinfe123) [reply packages/react-google-adk/src/AdkClient.ts]: Addressed in 965c3e230. The guard now matches the sibling adapters and handles both null and undefined bodies. A regression test uses a non-standard Response with an undefined body and confirms the stable adapter error.
